packages/module/src/SourcesCard/SourcesCard.scss

```diff
@@ -0,0 +1,87 @@
+.pf-chatbot__source {
+  display: flex;
+  flex-direction: column;
+  gap: var(--pf-t--global--spacer--sm);
+  padding-block-start: var(--pf-t--global--spacer--sm);
+  width: 22.5rem;
+}
+
+.pf-chatbot__sources-card {
+  box-shadow: var(--pf-t--global--box-shadow--X--sm--default) var(--pf-t--global--box-shadow--Y--sm--default) 4px 0px
+    #00000029;
+}
+
+.pf-chatbot__sources-card-title {
+  display: block;
+  display: -webkit-box;
+  height: var(--pf-v6-c-card__title-text--LineHeight);
+  -webkit-line-clamp: 1;
+  line-clamp: 1;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
rather than hardcoding ellipses here, could we wrap the Title in a Truncate component? https://www.patternfly.org/components/truncate. It'll include a little extra a11y out of the box.